decimal.go

```diff
@@ -1553,7 +1553,7 @@ func (d Decimal) Sin() Decimal {
 		sign = !sign
 		j -= 4
 	}
-	z := d.Sub(y.Mul(PI4A)).Sub(y.Mul(PI4B)).Sub(y.Mul(PI4C)) // Extended precision modular arithmetic
+  	z := d.Sub(y.Mul(PI4A)).Sub(y.Mul(PI4B)).Sub(y.Mul(PI4C)) // Extended precision modular arithmetic
```

Comment:
Could you revert this line? If this is gofmt issue, I will address this in separate PR.

Reply:
Yes, I trying to revert it right now